pkg/controller/installation/typha_autoscaler.go

```diff
@@ -156,15 +198,18 @@ func (t *typhaAutoscaler) autoscaleReplicas() error {
 	if err != nil {
 		return fmt.Errorf("could not get number of nodes: %w", err)
 	}
+	log.V(1).Info("Number of nodes to consider for typha autoscaling", "all", allSchedulableNodes, "linux", linuxNodes)
```

Comment:
I'm assuming these are just for testing as getting this message every 10 seconds seems a bit verbose.

Reply:
yeah these were just test logs